src/device/mod.rs

```diff
@@ -1,18 +1,40 @@
+use std;
+
 use manager;
 use manager::NetworkManager;
 
 
 #[derive(Debug)]
 pub struct Device {
-    interface: String,
-    path: String,
-    device_type: DeviceType,
-    state: DeviceState,
-    real: bool,
+    pub interface: String,
+    pub path: String,
```

Comment:
My thinking is that we better avoid using the dbus-rs types outside of the manager module, since working with them is awkward and painful. Paths are strings and using String directly is a good fit, but please let me know if you see any disadvantage.